src/jit/gentree.cpp

```diff
@@ -1078,7 +1078,15 @@ bool GenTreeCall::AreArgsComplete() const
         assert((gtCallLateArgs != nullptr) || !fgArgInfo->HasRegArgs());
         return true;
     }
+    
+#if FEATURE_FASTTAILCALL
+    // If the args are not complete and fgArgInfo is not null. Then most
```

Comment:
This comment should be reworded (partly because there should be a comma instead of a period after 'null'). I think this would be clearer:
```
// If we have FEATURE_FASTTAILCALL, 'fgCanFastTailCall()' can call 'fgInitArgInfo()', and in that
// scenario it is valid to have 'fgArgInfo' be non-null when 'fgMorphArgs()' first queries this,
// when it hasn't yet morphed the arguments.